google/cloud/bigtable/data/exceptions.py

```diff
@@ -311,3 +311,11 @@ def __init__(
         self.__cause__ = cause
         self.index = failed_index
         self.query = failed_query
+
+
+class InvalidExecuteQueryResponse(core_exceptions.GoogleAPICallError):
+    """Exception raised to invalid query response data from back-end."""
+
+
+class ParameterTypeInferenceFailed(ValueError):
+    pass
```

Comment:
nit: it would be good to have a docstring here

Reply:
done